fluent_contents/admin/placeholderfield.py

```diff
@@ -27,6 +29,16 @@ class PlaceholderFieldAdmin(PlaceholderEditorAdmin):
     """
     placeholder_inline = PlaceholderFieldInline
 
+    def get_form(self, request, obj=None, **kwargs):
+        kwargs['formfield_callback'] = partial(
+            self.formfield_for_dbfield, request=request, obj=obj)
```

Comment:
It might not be fully PEP8, but I'm fine with keeping a slightly longer line length (100 or 120 chars)
Also, it would be good to comment here for `formfield_callback` is redefined here so obj is also passed.